<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWidgetClass</class>
 <widget class="QWidget" name="AppWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="capBox">
     <property name="title">
      <string>Capture - Select Region</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="xOffsetLabel">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="left">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="regionWidthLabel">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="width">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yOffsetLabel">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="top">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="regionHeightLabel">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="height">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yOffsetLabel_2">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="frequency">
        <property name="suffix">
         <string> Hz</string>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QCheckBox" name="onTop">
        <property name="text">
         <string>Always on top</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="display1">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLabel" name="display2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Training</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Letters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="trainLetters">
        <property name="inputMask">
         <string>AAAAAA; </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="trainButton">
        <property name="text">
         <string>Train 6 syms</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOff">
       <property name="text">
        <string>Off</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGame">
       <property name="text">
        <string>Game</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChallenge">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Challenge</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLearn1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>A-Learn 1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLearn2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>A-Learn 2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Game Settings</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="xOffsetLabel_2">
        <property name="text">
         <string>Min Letters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="minLetters">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="xOffsetLabel_3">
        <property name="text">
         <string>Max Letters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="maxLetters">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Recognized</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="currentLetters">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QPlainTextEdit" name="words"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="allWords">
        <property name="text">
         <string>All words</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
